spring-cloud-task-samples/partitioned-batch-job/src/main/java/io/spring/JobConfiguration.java

```diff
@@ -104,7 +104,7 @@ public PartitionHandler partitionHandler(TaskLauncher taskLauncher, JobExplorer
 		partitionHandler.setCommandLineArgsProvider(new PassThroughCommandLineArgsProvider(commandLineArgs));
 		partitionHandler.setEnvironmentVariablesProvider(new SimpleEnvironmentVariablesProvider(this.environment));
 		partitionHandler.setMaxWorkers(1);
-		partitionHandler.setApplicationName("PartitionedBatchJobTask");
```

Comment:
I added a `toLowerCase()` for the id generated in the k8s deployer, so if you prefer, you can keep the name as it was